keyboards/alps64/keymaps/via/rules.mk

```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = no
```

Comment:
```suggestion
```

Reply:
I added this one and the options below, because the firmware wouldn't compile without removing those functions. I'll check if it works if I apply your suggestion and I'll come back with the results.